.github/workflows/check-pr-sync.yml

```
@@ -0,0 +1,32 @@
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  check-sync:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)